Задание_3/Задание_3/Program.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+
+namespace Задание_3
+{
+    internal class Program
+    {
+
+
+        static void Main()
+        {
+            double x, x3, x2, Y, A;
```

Comment:
Эти переменные, кроме `x` и `y` не нужны.
```suggestion
            double x, y;
```
Локальные переменные принято называть с маленькой буквы (не `Y`, а `y`).